### DISCUSSION TITLE
微軟 Designer 新 AI 製圖工具，輸入文字自動產生圖片

### DISCUSSION THREAD

**Discussion by ching020721** [1101-GS4719-C/Python-Community-GS4719-C#787] [科技新知]
https://www.bnext.com.tw/article/68498/dall-e-2-image-generator

微軟宣布365服務會增加Designer人工智慧創作工具，新功能是利用非營利AI研究組織Open AI開發的「DALL-E 2」影像生成 AI，可根據輸入文字製作插圖，不需要高超的繪圖技術便能生成漂亮的繪畫作品。DALL-E是OpenAI去年推出的電腦視覺系統CLIP作為基礎開發的製圖模型，OpenAI將圖像壓縮成一系列單字，並學習去預測接下來會出現什麼，2更是產出更逼真的圖像，甚至還能編輯圖片。但Deepfake和AI製圖的濫用所帶來的風險，目前製作出來的圖片都要人工審核，希望未來能夠更加進步並完善DALL-E 2的安全性。

其他DALL-E資訊：https://www.bnext.com.tw/article/68498/dall-e-2-image-generator

**Comment by Tuoresa** [comment]
好驚訝竟然能夠真的把文字轉換為圖像，雖然版權與各方面安全疑慮仍然尚未解決，但這項技術真的非常卓越也很有趣，感覺在成熟之後可以為各方面藝術創作及應用開啟新的里程碑！

**Comment by Pietra1226** [comment]
這樣之後說不定也能將聲音轉換為圖像，這樣溝通起來有聲音有圖就更方便清楚了

**Comment by Doreen331** [comment]
原本以為需要人類的創意（例如繪畫或創作音樂等等）的工作不會被AI取代，沒想到AI居然會創作了，說不定之後寫歌的人也會失業，只要輸入想要的曲風以及意境就可以創作出一首歌（透過機器學習好像也不是不可能）

**Comment by ruthchen1020** [comment]
不能否認AI真的很了解人類喜歡甚麼，對風格的拿捏也很準確，但說創意跟情感還是人類的藝術品才能展示。

**Comment by StevenWanglolz** [comment]
希望這個世代的藝術家可以撐下去，不要被這些科技取代掉!

**Comment by wenchi9023** [comment]
根據文字創造圖像已經很厲害了，竟然還能編輯! 如果這項功能能夠從2D拓展到3D，那麼運用的情景又更多了! 例如使用者描述想要這個空間如何，就能產出相對應適合該空間的3D圖等。但如此一來許多職業也會慢慢被取代，該如何使用這項技術，且能夠運用到多深入是另一個該省思的議題。
